src/Microsoft.ComponentDetection.Detectors/rust/DependencySpecification.cs

```diff
@@ -1,9 +1,9 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using Microsoft.ComponentDetection.Detectors.Rust.Contracts;
-using Semver;
 
-using Range = Microsoft.ComponentDetection.Detectors.Rust.SemVer.Range;
+using Range = SemanticVersioning.Range;
+using Version = SemanticVersioning.Version;
```

Comment:
Is this required? Can the imports be satisfied with `using SemanticVersioning;` instead?

Reply:
I tried that, but there is an ambiguous reference error.
![image](https://user-images.githubusercontent.com/8052624/220463149-9064cb32-8052-42d5-8a96-fbaa28667b6d.png)